experimenter/experimenter/templates/experimenter_base.html

```diff
@@ -8,9 +8,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{% block title %}{% endblock %}</title>
     <script src="{% static 'fontawesomefree/js/all.min.js' %}"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
Shouldnt this be the {% tailwindcss %} template tag from your other PR?

Reply:
So this is because the other PR was causing integration tests to fail and I was trying to figure out the reason so to get the UI changes reviewed earlier I used a CDN. I'll be replacing this once the other PR is merged, sorry

Reply:
No need to apologize, I was just confused :)